packages/features/bookings/components/AvailableTimes.tsx

```diff
@@ -67,11 +68,12 @@ const SlotItem = ({
 
   const offset = (usersTimezoneDate.utcOffset() - nowDate.utcOffset()) / 60;
 
-  const { isOverlapping, overlappingTimeEnd, overlappingTimeStart } = useCheckOverlapWithOverlay(
-    computedDateWithUsersTimezone,
-    selectedDuration,
-    offset
-  );
+  const { isOverlapping, overlappingTimeEnd, overlappingTimeStart } = useCheckOverlapWithOverlay({
+    start: computedDateWithUsersTimezone,
+    selectedDuration: event?.length ?? 0,
```

Comment:
We should never hit event.length being null here -> we pull directly from cache from the inital call from loading booker

Reply:
is 0 a safe response? I'm not sure what our slot logic does if it has a duration of 0

Reply:
0 is just for calculating overlap with overlay calendar - will just show as everything has available until something loads